composer.json

```diff
@@ -4,13 +4,19 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.0",
-        "laravel/framework": "5.5.* || 5.6.*"
+        "illuminate/database": "5.5.* || 5.6.*",
+        "illuminate/events": "5.5.* || 5.6.*",
+        "illuminate/support": "5.5.* || 5.6.*",
+        "php": "^7.0"
     },
     "authors": [
         {
             "name": "Francisco Neves",
             "email": "hi@francisconeves.me"
+        },
+        {
+            "name": "Szabolcs (Szasza) Palmer",
```

Comment:
I thank you so much for your PR, however I see unpractical to add all contributors to this file.

I will add credits to the README, pointing to contributors.

Reply:
Removed.